packages/components/organisms/f-checkout/src/components/Header.vue

```diff
@@ -23,9 +25,9 @@
         <div
             v-else
             data-test-id="guest-checkoutHeader">
-            <h2 :class="$style['c-checkoutHeader-title']">
+            <h1 :class="$style['c-checkoutHeader-guestTitle']">
```

Comment:
Could we add a `data-test-id="user-title"` attribute here too?

Reply:
I'll add it now